.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       # and hyphens.
       run: echo "TAG=branch-${GITHUB_REF_NAME//[^A-Za-z0-9._-]/-}" | tee -a $GITHUB_ENV
 
-    - run: ./devel/build $TAG
+    - run: ./docker/build $TAG
```

Comment:
Hmm. Having a directory called `docker` inside the `docker-base` repo seems a bit arbitrary to me. This whole repo is about Docker, so why are some things in an additional `docker` directory?

What about opting to leave the **devel**opment programs (`push`, `pull`, etc) in `devel/` and instead moving the builder programs (`download-repo`, `latest-augur-release-tag`) into a new `builder/` directory?

Reply:
Updated: 0770e9e048e18273d2ad0d08740ff3519c2cb8c1